bitnami/kubeapps/templates/_helpers.tpl

```diff
@@ -357,11 +357,11 @@ kubeapps: ingress.tls
       {{- range $plugin, $options := .Values.packaging }}
         {{- if $options.enabled }}
           {{- if eq $plugin "carvel" }}
-            {{- $enabledPlugins = append $enabledPlugins "kapp-controller" }}
+            {{- $enabledPlugins = append $enabledPlugins "kapp-controller-packages" }}
           {{- else if eq $plugin "flux" }}
-            {{- $enabledPlugins = append $enabledPlugins "fluxv2" }}
+            {{- $enabledPlugins = append $enabledPlugins "fluxv2-packages" }}
           {{- else if eq $plugin "helm" }}
-            {{- $enabledPlugins = append $enabledPlugins "helm" }}
+            {{- $enabledPlugins = append $enabledPlugins "helm-packages" }}
```

Comment:
This is the only functional change, requiring these packaging plugins to be present in a directory suffixed with `-packages`.

Reply:
Is not the `resources` plugin going to be modified in the same way? Is defined some lines below as
```
{{- $enabledPlugins = append $enabledPlugins "resources" }}
```

Reply:
Nope. The resources plugin isn't a packages plugin. The suffix is just to ensure that the plugins that are for providing packages are marked as such (so, for example, we can later differentiate and have a fluxv2-kustomize or similar). Funnily, it was your original commit that made us realise we should do that, so we updated in our development branch after the last release. You'll see that the commit in the pipeline which reverted the change did not need to affect resources because you'd not had a suffix for that.

Thanks for the review!

Reply:
Understood, thanks for the clarification!